src/app/main.py

```diff
@@ -44,16 +45,20 @@
 # HydroLAKES
 lakes = ee.FeatureCollection('users/gena/HydroLAKES_polys_v10')
 
+
+# available datasets
+bathymetry = {
```

Comment:
Might be better to merge this with other raster datasets currently listed in get_raster, so that we will have just two functions, to download raster dataset and to get tile url to visualize rasters (providing style or having it pre-configured per raster).